server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/DraftMethodsModule.java

```diff
@@ -43,10 +43,18 @@
 import org.apache.james.jmap.draft.methods.SetMessagesProcessor;
```

Comment:
The comment title is IMO wrong (`JAMES-3093 Backport Authenticator in jmap-rfc8621 module `)

I should be `JAMES-3093 Move authenticator to JMAP common module`

And a quick explanation for the "why" could be welcome too. I propose:

```
Each JMAP version can propose its own AuthenticationStrategies, bundled in their own Authenticator, but the way Authentication is performed is common to the two versions.
```